### PR TITLE
Migration step 22

### PR DESCRIPTION
**Pull request by Nicoss54**
#104 

migration step 22 to 24

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -68,12 +68,14 @@
     "variable-name": false,
     "whitespace": [true, "check-branch", "check-decl", "check-operator", "check-separator", "check-type"],
     "no-output-on-prefix": true,
-    "directive-selector": [true, "attribute", "sfeir", "camelCase"],
+    "directive-selector": [true, "attribute", "sfeir", "camelCase", "kebab-case"],
```
Comment (noelmace) [inline tslint.json]: I think it's a bad practice. Why did you add that? If it's because of an existing use case, I think fixing it instead of changing the rules would be better.

File: tslint.json [hunk 2]
```diff
@@ -68,12 +68,14 @@
...
     "component-selector": [true, "element", "sfeir", "kebab-case"],
     "use-input-property-decorator": true,
     "use-output-property-decorator": true,
     "use-host-property-decorator": true,
-    "no-input-rename": true,
+    "no-input-rename": {
+      "severity": "warn"
```
Comment (noelmace) [inline tslint.json]: same thing as above: I think it's a bad practice. Fixing the use cases instead of changing the rules would be better.

File: package-lock.json [hunk 3]
```diff
@@ -795,15 +795,10 @@
         }
       }
     },
-    "@ngrx/router-store": {
```
[the remark this answers is not in the document]
Comment (Nicoss54) [reply package-lock.json]: No it isn't used on this project

**Comment by noelmace** [comment]
Please make all the unit tests in steps/unit-tests fail by adding a expect(true).toBe(false) and update the slides accordingly, making the goal of the exercise simpler to understand: "just fix the tests" (with a step by step of course).

File: package.json [hunk 4]
```diff
@@ -48,6 +48,8 @@
     "@angular/cli": "7.0.6",
     "@angular/compiler-cli": "~7.1.0",
     "@angular/language-service": "~7.1.0",
+    "@ngxs/devtools-plugin": "^3.3.2",
+    "@ngxs/logger-plugin": "^3.3.2",
```
[the remark this answers is not in the document]
Comment (Nicoss54) [reply package.json]: done

File: angular.json [hunk 5]
```diff
@@ -4345,6 +4345,895 @@
           }
         }
       }
+    },
+    "toto": {
```
[the remark this answers is not in the document]
Comment (Nicoss54) [reply angular.json]: done, sorry for that

File: angular.json [hunk 6]
```diff
@@ -4345,6 +4345,895 @@
...
+              "steps/ngrx-solution/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**"
+            ]
+          }
+        }
+      }
+    },
+    "ngrx-solution-e2e": {
```
[the remark this answers is not in the document]
Comment (Nicoss54) [reply angular.json]: done

**Comment by Nicoss54** [comment]
> Please make all the unit tests in steps/unit-tests fail by adding a expect(true).toBe(false) and update the slides accordingly, making the goal of the exercise simpler to understand: "just fix the tests" (with a step by step of course).

done too ;)
